### PR TITLE
Add ethernet default cost parameter to models

### PR DESCRIPTION
**Pull request by gkodali-zededa**
Signed-off-by: gkodali-zededa <gkodali@zededa.com>

### CODE REVIEW

File: models/template_l1_FWA-1012VC.json [hunk 1]
```diff
@@ -111,6 +118,7 @@
         "PciLong": "0000:00:15.0"
       },
       "logicallabel": "wwan0",
+      "cost": 0,
```
Comment (eriknordmark) [inline models/template_l1_FWA-1012VC.json:121]: We shouldn't default this to zero for wwan

File: models/template_l1_SYS-E100-9APP-wwan.json [hunk 2]
```diff
@@ -49,6 +51,7 @@
         "PciLong": "0000:00:15.0"
       },
       "logicallabel": "wwan0",
+      "cost": 0,
```
Comment (eriknordmark) [inline models/template_l1_SYS-E100-9APP-wwan.json:54]: Ditto

**Review by eriknordmark — CHANGES_REQUESTED** [review]
Please pick a non-zero number than zero for wwan0/wwan1.
How about using 10 for those?

File: models/template_l1_SYS-E100-9APP-wwan.json (hunk 2 above)
Comment (gkodali-zededa) [reply models/template_l1_SYS-E100-9APP-wwan.json]: fixed. Made it 10

File: models/template_l1_FWA-1012VC.json (hunk 1 above)
Comment (gkodali-zededa) [reply models/template_l1_FWA-1012VC.json]: fixed. Made it 10

**Comment by gkodali-zededa** [comment]
> Please pick a non-zero number than zero for wwan0/wwan1.
> How about using 10 for those?

Done. Made it 10

**Review by eriknordmark — APPROVED** [review]
LGTM

**Pull request merged**